web/packages/teleterm/src/ui/ConnectMyComputer/DocumentConnectMyComputerStatus/DocumentConnectMyComputerStatus.story.tsx

```diff
@@ -43,7 +46,45 @@ export function NotStarted() {
 }
 
 export function Running() {
-  return <ShowState agentProcessState={{ status: 'running' }} />;
+  const appContext = new MockAppContext({ appVersion: '17.0.0' });
+
+  let agentUpdateListener: (state: AgentProcessState) => void;
+  appContext.mainProcessClient.subscribeToAgentUpdate = (
+    rootClusterUri,
+    listener
+  ) => {
+    agentUpdateListener = listener;
+    return { cleanup: () => undefined };
+  };
```

Comment:
I don't like this approach but I see no other way of making it work from within a story.